Assets/Scripts/Controllers/States/CharacterQueries.cs

```diff
@@ -0,0 +1,5 @@
+public static class CharacterQueries
```

Comment:
What are your thoughts on moving this into the Models folder alongside MouseCodes?

Or, how about if we made a new folder called Helpers at the same level as the Models/Views/Controllers directories, with these inside it?

Reply:
I think making a Helpers folder would be a good idea, I'll do that.